teamA/lib/Views/dashboard.dart

```diff
@@ -0,0 +1,279 @@
+                      style: TextStyle(
+                        fontSize: 14,
+                        color: Theme.of(context).colorScheme.onBackground,
+                      ),
+                    ),
+                    SizedBox(height: 10),
+                    _buildDashboardButton(
+                        context, 'Courses', 150), // Smaller button
+                  ],
+                ),
+                SizedBox(width: 20),
+                Column(
+                  children: [
+                    Text(
+                      'Teacher creates/views assessments.',
+                      style: TextStyle(
+                        fontSize: 14,
+                        color: Theme.of(context).colorScheme.onBackground,
+                      ),
+                    ),
+                    SizedBox(height: 10),
+                    _buildDashboardButton(
+                        context, 'Assessments', 180), // Larger middle button
+                  ],
+                ),
+                SizedBox(width: 20),
+                Column(
+                  children: [
+                    Text(
+                      'Teacher can view, grade, and create essays.',
+                      style: TextStyle(
+                        fontSize: 14,
+                        color: Theme.of(context).colorScheme.onBackground,
+                      ),
+                    ),
+                    SizedBox(height: 10),
+                    _buildDashboardButton(
+                        context, 'Essays', 150), // Smaller button
+                  ],
+                ),
+              ],
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  // Mobile layout
+  Widget _buildMobileLayout(BuildContext context) {
+    return Center(
+      child: SingleChildScrollView(
+        padding: const EdgeInsets.all(16.0),
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            Text(
+              'Teacher Dashboard',
+              style: TextStyle(
+                fontSize: 20,
+                fontWeight: FontWeight.normal,
+                color: Theme.of(context).colorScheme.onBackground,
+              ),
+            ),
+            SizedBox(height: 20), // Space between the title and buttons
+            Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Column(
+                  children: [
+                    Text(
+                      'View available courses.',
+                      style: TextStyle(
+                        fontSize: 14,
+                        color: Theme.of(context).colorScheme.onBackground,
+                      ),
+                    ),
+                    SizedBox(height: 10),
+                    _buildDashboardButton(
+                        context, 'Courses', 120), // Smaller for mobile
```

Comment:
Too small to fit the text. Either reduce font size or make the buttons bigger.

Reply:
resolved